manifests/s/SystweakSoftware/AdvancedDiskRecovery/2.7.1200.18511/SystweakSoftware.AdvancedDiskRecovery.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Automatically updated by the winget bot at 2022/Nov/26
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: SystweakSoftware.AdvancedDiskRecovery
+PackageVersion: 2.7.1200.18511
+PackageLocale: en-US
+Publisher: Systweak Software
+PackageName: Advanced Disk Recovery
+License: Systweak Software

```

```suggestion
License: Proprietary
```
Need a valid license here.